grammar/Readme.md

```diff
@@ -1,25 +1,10 @@
 Note: The file `JavaScript.tmLanguage.json` is derived from [TypeScriptReact.tmLanguage](https://github.com/Microsoft/TypeScript-TmLanguage/blob/master/TypeScriptReact.tmLanguage).
 
 # To update the grammar after making changes:
+
```

Comment:
Stray newline

Reply:
It's pretty common to have a new line after a header and before a list so this should be fine to merge as-is